example_site/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": false,
   "scripts": {
-    "start": "BROWSER_ARGS=--enable-precise-memory-info react-scripts start",
+    "start": "react-scripts start",
```

Comment:
You don't need to commit this change - that was my bad!

I did not have time yesterday to fully look into what was going on with your machine, but `BROWSER_ARGS` is an [advanced configuration](https://create-react-app.dev/docs/advanced-configuration/) for npm that does not seem to be playing nicely with your current setup. My guess is that is has something to do with the windows machine vs. the mac. This option is ignored during production so it's not a huge deal, but until we get your laptop to you I think you'll need to make this change locally before you start developing. 

@scottanderson42 might have a little more insight. When Jenny tried to run `npm start` with  `BROWSER_ARGS` in the command she got the following error: 
![image](https://user-images.githubusercontent.com/22031072/121520187-a2f80880-c9c0-11eb-95b7-023a638c09b9.png)